docs/xaml/fundamentals/get-started.md

```diff
@@ -54,6 +54,13 @@ The `MainPage` class derives from <xref:Microsoft.Maui.Controls.ContentPage>, an
 
 When Visual Studio builds the project, a source generator generates new C# source that contains the definition of the `InitializeComponent` method that's called from the `MainPage` constructor and adds it to the compilation object.
 
+::: moniker range=">=net-maui-10.0"
+
+> [!IMPORTANT]
+> XAML source generation is a preview feature in .NET 10. To enable it, set `<EnablePreviewFeatures>true</EnablePreviewFeatures>` in your project and add `[assembly: XamlProcessing(XamlInflator.SourceGen)]` to your code. Preview features may change before GA.
```

Comment:
no longer accurate. the EnablePreview is no longer necessary (but I can make it so for rc2), and you turn it on by msbuild metadata

```
<ItemGroup>
    <MauiXaml Update="MyPage.xaml" Inflator="SourceGen" />
</ItemGroup>
```

you can obviously wildcard it "**\*.xaml"